ref/src/CMakeLists.txt

```diff
@@ -16,5 +22,5 @@ if(OpenMP_FOUND)
     target_link_libraries(${PROJECT_NAME} PUBLIC OpenMP::OpenMP_C)
 endif()
 
-install(TARGETS yppm
+install(TARGETS yppm interpolateTest
```

Comment:
Installation should not include the test code.  Installation is for production, not testing.

Reply:
Done.